### ISSUE TITLE
relative import error

### ISSUE THREAD

**Issue by simon-q** [vojtajina/sublime-OpenRelated#9]
On w7, portable sublime2.0.1 build 2217
i experienced error while loading open related plugin.

```
Reloading plugin C:\Develop\Sublime Text 2\Data\Packages\Open Related\converter_test.py
Traceback (most recent call last):
  File ".\sublime_plugin.py", line 62, in reload_plugin
  File ".\converter_test.py", line 2, in <module>
    from .converter import *
ValueError: Attempted relative import in non-package

Reloading plugin C:\Develop\Sublime Text 2\Data\Packages\Open Related\open_related.py
Traceback (most recent call last):
  File ".\sublime_plugin.py", line 62, in reload_plugin
  File ".\open_related.py", line 3, in <module>
    from . import converter
ValueError: Attempted relative import in non-package
```

Started to work after reverting changes in commit 93e4bbb in files:
converter_test.py
open_related.py

**Comment by gerolfpic** [comment]
I have the same problem.

**Comment by vojtajina** [comment]
I think this is ST2 issue, I'm closing it as I believe people already switched to ST3.

If you really want to fix this, check out #10 and use that branch, or update the PR so that it works with both ST2/3.
